docs/tutorial/maxcut.ipynb

```diff
@@ -132,7 +132,7 @@
     "The Max-Cut problem can be formulated with the following equation:\n",
```

Comment:
It might be better to first show the model with the maximization problem, and then explain that it is turned into a minimization problem by multiplying by -1.